rust/src/cli.rs

```diff
@@ -48,6 +56,10 @@ pub struct Args {
     /// Available subcommands
     #[command(subcommand)]
     pub command: Option<Command>,
+
+    /// Optional CloudFormation stack to lookup key and bucket
+    #[arg(long, env)]
```

Comment:
Having the arguments be read from env only in the CLI and not the lib breaks compatibility with the Python version, where the library version also tries to read from env (correct me if I'm wrong on this, but this is what I understood from reading the Python).

I'm personally in favor of implementing this way but question is, do we want to keep the interface just as a potential extension of the Python one or just have it be different?